test/views/CreditAndRefundsViewSpec.scala

```diff
@@ -51,6 +51,12 @@ class CreditAndRefundsViewSpec extends TestSupport with FeatureSwitching with Im
   val creditAndRefundFromHMRCTitlePart1: String = messages("credit-and-refund.credit-from-hmrc-title-prt-1")
   val creditAndRefundFromHMRCTitlePart2: String = messages("credit-and-refund.credit-from-hmrc-title-prt-2")
   val creditAndRefundPaymentFromEarlierYearLinkText: String = messages("paymentHistory.paymentFromEarlierYear")
+  val creditAndRefundAgentHasCredits2: String = messages("credit-and-refund.agent.subHeading.has-credits-2")
```

Comment:
if variable use only once, maybe we don't need it all ? you can call messages(..) directly where required etc